docs/shared/tutorials/react-monorepo.md

```diff
@@ -835,41 +846,41 @@ npx nx graph --affected
 
 {% /graph %}
 
-## Building the Apps for Deployment
+## Build the Apps for Deployment
 
 {% video-link link="https://youtu.be/gc4N7kxiA50?t=713" /%}
 
 If you're ready and want to ship your applications, you can build them using
 
 ```{% command="npx nx run-many -t build" path="react-monorepo" %}
-vite v4.3.5 building for production...
-✓ 33 libs transformed.
-dist/react-store/index.html                   0.48 kB │ gzip:  0.30 kB
-dist/react-store/assets/index-e3b0c442.css    0.00 kB │ gzip:  0.02 kB
-dist/react-store/assets/index-378e8124.js   165.64 kB │ gzip: 51.63 kB
-✓ built in 496ms
+   ✔  nx run @react-monorepo/inventory:build (841ms)
+   ✔  nx run @react-monorepo/react-store:build (968ms)
 
-——————————————————————————————————————————————————————————————————————————————————————————————————————————
+—————————————————————————————————————————————————————————————
 
-NX   Successfully ran target build for project reactutorial (1s)
+ NX   Successfully ran target build for 2 projects (1s)
 ```
 
-All the required files will be placed in `dist/react-store` and `dist/inventory` and can be deployed to your favorite hosting provider.
+All the required files will be placed in `/apps/react-store/dist` and `/apps/inventory/dist` and can be deployed to your favorite hosting provider.
 
 You can even create your own `deploy` task that sends the build output to your hosting provider.
```

Comment:
I don't think we should say `You can even create your own` since everyone knows about package scripts. Maybe something like, "Nx can run your package scripts...".

Also, I think the expanded targets + scripts is more bloated, maybe we can show just the script, then show that you can add the `targets` entry to specify dependsOn. We can also note that they can be combined into `command` under `targets`.